hidapi.podspec

```diff
@@ -0,0 +1,30 @@
+Pod::Spec.new do |spec|
+
+  spec.name         = "hidapi"
+  spec.version      = "0.9.1"
```

Comment:
I used minor version increment, but I can change this value if the version of the next release is known.

Reply:
is there a way to read version from file? I'd rather keep it in ove place
also, I think next will be 0.10.0 - we likely will add some new API

Reply:
@Youw it's possible, but we have to decide file name. I can add `VERSION` file and fix podspec to use version from this file

Reply:
`VERSION` is fine
please do

Reply:
@Youw Unfortunately I was wrong. We have to push only one file to cocoapods, so we can't read version from separate file. Good news, we don't need to release new version to distribute 0.9.0 version. Version in cocoapods file just a link to github tag.